settings/common.py

```diff
@@ -33,6 +33,7 @@
 
 ALLOWED_HOSTS = []
 
+API_HOST_URL = ""
```

Comment:
Can we please get it from the environment and if the environment isn't having then use the default value `http://localhost:8000`